src/updater.cc

```diff
@@ -42,18 +55,21 @@ PollingMetadataUpdater::~PollingMetadataUpdater() {
   }
 }
 
-void PollingMetadataUpdater::start() {
+bool PollingMetadataUpdater::ValidateConfiguration() const {
+  return period_ > seconds::zero();
+}
+
+void PollingMetadataUpdater::StartUpdater() {
   timer_.lock();
   if (config().VerboseLogging()) {
     LOG(INFO) << "Timer locked";
   }
-  if (period_ > seconds::zero()) {
```

Comment:
Still need this, because we want `0` to turn off polling without an error.

Reply:
Done.